docs/topics/request-response.rst

```diff
@@ -581,7 +581,7 @@ Sending a JSON POST request with a JSON payload::
 Response objects
 ================
 
-.. class:: Response(url, [status=200, headers=None, body=b'', flags=None, request=None])
+.. autoclass:: Response
```

Comment:
could you please separate Response doc changes to a different PR, to allow merging them separately?

Reply:
Done, please see #4055. Thanks!